model-optimizer/extensions/back/LeakyReLUMutation.py

```diff
@@ -39,11 +39,12 @@ def pattern():
     @staticmethod
     def replace_pattern(graph: Graph, match: dict):
         relu = match['leakyrelu']
+        relu_name = relu.soft_get('name', relu.id)
         if not relu.has_valid('negative_slope'):
             return
         # Create PReLU op and reconnect input/output from LeakyReLU to PReLU
-        prelu = PreluOp(graph, dict(name=relu.name)).create_node()
-        const = Const(graph, dict(name=relu.name + "/weights", value=np.array([relu.negative_slope]))).create_node()
+        prelu = PreluOp(graph, dict(name=relu_name)).create_node()
+        const = Const(graph, dict(name=relu_name + "/weights", value=np.array([relu.negative_slope]))).create_node()
```

Comment:
Need to call `rename_nodes` to save the original name of the operation `relu` to the `prelu`

Reply:
Done.